Applications/ProductRatings/jni/main.cpp

```diff
@@ -247,8 +247,8 @@ int main(int argc, char *argv[]) {
 
       try {
         float answer =
-          NN.inference({MAKE_SHARED_TENSOR(nntrainer::Tensor({o}))})[0]
-            ->apply(stepFunction)
+          NN.inference({MAKE_SHARED_TENSOR(nntrainer::Tensor({o}, nntrainer::TensorDim::TensorType()))})[0]
```

Comment:
```suggestion
          NN.inference({MAKE_SHARED_TENSOR(nntrainer::Tensor({o}, {nntrainer::Tformat::NCHW, nntrainer::Tdatatype::FP32}))})[0]
```

Reply:
the default value of TensorType is 
```c++
{nntrainer::Tformat::NCHW, nntrainer::Tdatatype::FP32}
```
as in tensor_dim.h